src/init/omp.fish

```diff
@@ -1,6 +1,7 @@
 set -g posh_theme ::CONFIG::
 
 function fish_prompt
+    set stack_count 0 #(dirs -p | wc -l)
```

Comment:
why not local?

Reply:
I was quickly testing and I forgot to add `- l`. It should count correctly on all supported shells now.